### ISSUE TITLE
error in running make_brainGraphList

### ISSUE THREAD

**Issue by lupbla** [cwatson/brainGraph#23]
I followed the brainGraph user guide for version 3.0.0., but when I get to the make_brainGraphList command I get the following error:
Error in { : task 1 failed - "invalid 'length.out' value"
This is a sample of the adjacency matrices
[adjmats.zip](https://github.com/cwatson/brainGraph/files/5491180/adjmats.zip)
Could you help fix this error?

**Comment by cwatson** [comment]
Sorry, I thought I had replied to this sooner. I need the code you used and all the data that would allow me to reproduce this error.

**Comment by lupbla** [comment]
Hi,
Thank you for your response.
I send the code I used, I tried to make it as straightforward as possible.
I think it's still messy, but I'm not very good at coding, sorry.
the first two files are the data and the last one has the scripts.

The graphs_old code kind of works, but when I try to get the results into
object g with script 04_save it gives me an error.
the makeGraphs code gives the length.out error
Thank you

**Comment by cwatson** [comment]
I still have not received any code. I cannot debug your issue without it.

**Comment by lupbla** [comment]
I apologize, I sent you a link to my Google Drive because I got a message
that the email was too big. The data is in the drive link:
https://drive.google.com/drive/folders/1RDCJ17Hgjzk4FSQ8_-5Ne71asHu0neC9?usp=sharing
and the scripts are in the zipped folder I attach.
Thank you

**Comment by cwatson** [comment]
Please do not use code from older versions, as `v3.0.0` is not backwards compatible. I have a few other comments:

- In your `03_makeGraphs.R` script, change the argument `groups` to `grpNames`.
- You set `weighting='sld'` but your modality is `fmri`; this does not fit as `sld` is specific to DTI (streamline density).
- I do not understand why you load so many packages in your initial script. I presume you use them for some purposes in another script, but it might not be necessary. It could introduce issues with function names conflicting across packages.

Otherwise, I did not get any errors. Please make sure you read through the *current* User Guide. Specifically *Chapter 7.3*; there is a code block that you can try and it should work for you.
If you are new to R programming/scripting, I recommend taking some kind of introductory (short) course or following some tutorials (which can be found online), or if you have the time to read an introductory book (there should be some free e-books out there somewhere).

I will close the issue for now, but please try my suggestions and if there continues to be a problem let me know. You may want to join the Google Group, as I think it is more appropriate for small errors such as this. But if you feel it is necessary, you may reopen the issue.

**Comment by lupbla** [comment]
thank you very much.
What should I set the weighting to instead?
I will look up for the google group

**Comment by lupbla** [comment]
I tried your suggestions, instead of using the power atlas as custom I used
yours and it looked like it was working, but when I check the g object I
get this
Error in seq_len(no) : argument must be coercible to non-negative integer
Is installing the previous version my only option?
I attach the code I used and the data is the same as I had shared
https://drive.google.com/drive/folders/1RDCJ17Hgjzk4FSQ8_-5Ne71asHu0neC9?usp=sharing
This is my session info

> sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 10 (buster)

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.8.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.8.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C
 [9] LC_ADDRESS=C               LC_TELEPHONE=C
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C

attached base packages:
[1] parallel  stats     graphics  grDevices utils     datasets  methods
[8] base

other attached packages:
[1] data.table_1.12.8 doMC_1.3.6        iterators_1.0.12  foreach_1.4.8
[5] brainGraph_3.0.0  igraph_1.2.4.2

loaded via a namespace (and not attached):
 [1] lattice_0.20-40   codetools_0.2-16  permute_0.9-5     MASS_7.3-51.5
 [5] grid_3.6.3        magrittr_1.5      doParallel_1.0.15 Matrix_1.2-18
 [9] tools_3.6.3       abind_1.4-5       compiler_3.6.3    pkgconfig_2.0.3

**Comment by cwatson** [comment]
I assume you are referring to the error you see when you simply type `g`. That error is specific to `igraph`, and it is not a critical error, so you would have to contact the `igraph` developers to solve it. However, it should not affect data analysis.

**Comment by lupbla** [comment]
thank you, I think I just panicked.

**Comment by lupbla** [comment]
I just noticed that when the make_brainGraphList command ran was when I set
atlas <- 'power264'
but I excluded five regions when I created the correlation matrix.
When I use
subnet= regions
and set regions to
the names of the excluded regions it gives me the same error
Error in { : task 1 failed - "invalid subscript type 'list'"

I just don't know how can the graphs be made if the atlas does not match
the dimensions of the matrix?
I suppose it's taking the first 259 regions and leaving the remaining five
out.
The code and data I used i exactly the same as the previous I sent you.
I'm really sorry to continue to ask about the same question, I was running
the previous version and it was working fine. When I saw the new release I
didn't think twice before updating.
